{
  Exception e=new AuthorizationException();
  assertEquals(Utils.returnFirstExceptionIfInListOrThrowStandardExceptionOrCause(new Class[]{},new RuntimeException(e)),e);
}
